pkgs/os-specific/bsd/freebsd/pkgs/mkDerivation.nix

```diff
@@ -89,6 +89,7 @@ lib.makeOverridable (
       # Since STRIP in `makeFlags` has to be a flag, not the binary itself
       STRIPBIN = "${stdenv'.cc.bintools.targetPrefix}strip";
     }
+    // lib.optionalAttrs (!stdenv.hostPlatform.isFreeBSD) { BOOTSTRAPPING = true; }
```

Comment:
```suggestion
    // lib.optionalAttrs (!stdenv.hostPlatform.isFreeBSD) { env.BOOTSTRAPPING = true; }
```

Reply:
That would override packages that set env themselves. If we're going to switch to env then everything should switch to it.

Reply:
> That would override packages that set env themselves.

Easy fix: `env = attrs.env // { BOOTSTRAPPING = true; }`.

> If we're going to switch to env then everything should switch to it.

True but that's not a reason to keep using a deprecated feature.

Reply:
I pushed an additional commit which moves all the environment variables in this file to env.